packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -355,7 +358,10 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 		this.verifyNotClosed();
 		if (this.isAttached()) {
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-			this.services!.deltaConnection.submit(content, localOpMetadata);
+			this.services!.deltaConnection.submit(
+				makeHandlesSerializable(content, this.serializer, this.handle),
```

Comment:
Note: `reSubmitCore` below will need to be updated as well, either to inline `submitLocalMessage` without this `makeHandlesSerializable` call, or to decode the handles first (but that seems like wasted CPU to round-trip; not worth it merely to share 2 straightforward lines of code)